### PR TITLE
Adapt TheiaCon announcement

### LINKED ISSUE #459: Adapt TheiaCon announcement

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed #459

### CODE REVIEW

**Comment by JonasHelming** [comment]
Whoever reviews this, if you approve, feel free to directly merge

File: src/components/index/Header.js [hunk 1]
```diff
@@ -100,12 +100,12 @@ const Header = () => (
                         <img className="banner__image" src={TheiaConLogo}></img>
                     </a>
                     <h1 style={{ fontSize: '2.2rem' }}>
-                        The Call for Presentations is now OPEN for the <a href="https://events.eclipse.org/2023/theiacon/">2023 TheiaCon Event</a>!
-                        Submit your proposal today to be a speaker at the largest Theia event of the year!
+                        <a href="https://events.eclipse.org/2023/theiacon/">TheiaCon 2023</a> is approaching! | VIRTUAL EVENT | November 15 - 16, 2023
+                        Get registered the largest Theia event of the year!
```
Comment (vince-fugnitto) [inline src/components/index/Header.js:104]:
```suggestion
                        Get registered for the largest Theia event of the year!
```

File: src/components/index/Header.js [hunk 2]
```diff
@@ -100,12 +100,12 @@ const Header = () => (
                         <img className="banner__image" src={TheiaConLogo}></img>
                     </a>
                     <h1 style={{ fontSize: '2.2rem' }}>
-                        The Call for Presentations is now OPEN for the <a href="https://events.eclipse.org/2023/theiacon/">2023 TheiaCon Event</a>!
-                        Submit your proposal today to be a speaker at the largest Theia event of the year!
+                        <a href="https://events.eclipse.org/2023/theiacon/">TheiaCon 2023</a> is approaching! | VIRTUAL EVENT | November 15 - 16, 2023
```
Comment (vince-fugnitto) [inline src/components/index/Header.js:103]: minor: do we want a `|` to separate the date, else it looks out of place?

```suggestion
                        <a href="https://events.eclipse.org/2023/theiacon/">TheiaCon 2023</a> is approaching! | VIRTUAL EVENT | November 15 - 16, 2023 | 
```

image

**Review by vince-fugnitto — APPROVED** [review]
LGTM :+1:

**Pull request merged**